onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2650,6 +2650,9 @@ static constexpr OrtApi ort_api_1_to_14 = {
     &OrtApis::UpdateEnvWithCustomLogLevel,
     &OrtApis::SetGlobalIntraOpThreadAffinity,
     &OrtApis::RegisterCustomOpsLibrary_V2,
+    // End of Version 14 - DO NOT MODIFY ABOVE (see above text for more information)
+
+    // Start of Version 15 API in progress, safe to modify/rename/rearrange until we ship
     &OrtApis::RegisterCustomOpsUsingFunction,
```

Comment:
These APIs have the comment `\since Version 1.14` in the [onnxruntime_c_api.h](https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h#L3692) header. If they're being excluded from 1.14, then we should probably update the comments.

Also, the rel-1.14.0 branch still lists these APIs under version 1.14.